transcom/mymove
# MB-16879 Add SIT FSCs to invoice generator

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16879)

## Summary

This tickets adds DOSFSC and DDSFSC to the invoice generator. 


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Create a testharness `HHGMoveInSITEndsYesterday` move and note the move code
2. Login as the prime and view the move
3. Create a payment request that includes `Domestic origin SIT fuel surcharge` and `Domestic destination SIT fuel surcharge`
4. Login as a TIO and view the move
5. Review the weights so that we can review the payment request
6. Approve both service items and authorize payment
7. Copy the payment request number 
8. In the terminal, run `go run ./cmd/generate-payment-request-edi/ --payment-request-number XXXX-XXXX-X` and make sure there are no errors

### Backend
